pom.xml

```diff
@@ -120,6 +120,7 @@
         <build.type>SNAPSHOT</build.type>
         <maven.build.timestamp.format>yyyyMMddHHmm</maven.build.timestamp.format>
         <build.qualifier>v${maven.build.timestamp}</build.qualifier>
+        <sonatypeOssDistMgmtStagingUrl/>
```

Comment:
Please put there `<properties><sonatypeOssDistMgmtStagingUrl>https://jakarta.oss.sonatype.org/content/repositories/staging/</sonatypeOssDistMgmtStagingUrl>` as this value is needed for snapshosts.